### PR TITLE
dev: the printer just needs Output configuration

### PR DESCRIPTION
**Pull request by ldez**
the printer doesn't need the full configuration, the Output configuration is enough.

### CODE REVIEW

**Review by alexandear — APPROVED** [review]

File: pkg/commands/run.go [hunk 1]
```diff
@@ -327,11 +328,24 @@ func (c *runCommand) runAndPrint(ctx context.Context, args []string) error {
...
+	c.printDeprecatedLinterMessages(enabledLintersMap)
+
 	issues, err := c.runAnalysis(ctx, args)
 	if err != nil {
 		return err // XXX: don't lose type
 	}
 
+	// Fills linters information for the JSON printer.
+	for _, lc := range c.dbManager.GetAllSupportedLinterConfigs() {
+		isEnabled := enabledLintersMap[lc.Name()] != nil
```
Comment (Antonboom) [inline pkg/commands/run.go:345]: more go way?
```
_, isEnabled := enabledLintersMap[lc.Name()]
```

**Review by Antonboom — COMMENTED** [review]
👍

File: pkg/commands/run.go (hunk 1 above)
Comment (ldez) [reply pkg/commands/run.go:345]: it was existing code.

Comment (ldez) [reply pkg/commands/run.go:345]: It's not the same thing: the key can exist but the value can be nil.

**Pull request merged**